.gitignore

```diff
@@ -15,6 +15,7 @@ atlantis
 atlantis.env
 *.act
 Dockerfile.local
+vendor/
```

Comment:
Is this necessary?
```suggestion
```

Reply:
Oh no! It’s not necessary. My muscle memory probably did go mod vendor in the early days of the PR and totally forgot. Will remove it later today.

Thank you!

Reply:
Actually I thought I had commit the vendor folder and not an ignore for it. I think it’s ok to leave it since we don’t do vendoring on this project.